toolchain/defs.bzl

```
@@ -78,29 +80,40 @@ def toolchains(
         url_formats = [mirror_format, original_format]
 
     zig_repository(
-        name = "zig_sdk",
+        name = "{}-zig_sdk".format(exec),
```

```suggestion
        name = "zig_sdk-{}".format(exec),
```

Nit: let's put the platform name after zig_sdk